department-of-veterans-affairs/caseflow
# Mkerns/caseflow 1708 ihp parent task

Resolves [1708](https://vajira.max.gov/browse/CASEFLOW-1708)

### Description
I updated the Track Veteran Task model  to update the IHP parent task to the distribution task if it exist or the root task if the distribution task does not exist.

### Acceptance Criteria
- [ ] When the parent task of a newly created IHP task is different than the original task's parent, we set the correct parent task in the task tree

### Testing Plan
1. Go to Case Details
2. Scroll down to the Power Of Attorney Details section and click "Refresh POA" Button.
3. Scroll down to the Caseflow Timeline and verify IHP Task has successfully been cancelled and reassigned.
